deploy/roles/deploy.builder/defaults/main.yml

```diff
@@ -21,6 +21,7 @@ sg_events_use_tls: "true"
 sg_events_tls_client_cert: /config/certs/admin-cert
 sg_events_tls_client_key: /config/certs/admin-key
 sg_events_tls_ca_cert: /config/certs/admin-ca
+sg_events_tls_server_name: elasticsearch.sa-telemetry.svc
```

Comment:
So ultimately I _should_ have probably opened a separate pull request for this since it's not entirely related to the spirit of this PR. Adding this was based on an attempt to resolve an issue with connecting the SG to ES and it returning an error about invalid x509 certs. The actual issue was larger, because the import process for building the downstream container image was significantly broken. That has now been resolved and I'll be writing up additional documentation downstream to account for the new (easier!) import and release process.

However I'm leaving this here because I still think it's the more correct configuration when connecting the SG to ES since having this parameter set results in the SG not getting configured for `tlsConfig.InsecureSkipVerify`.